scripts/crypto/ibe-bn254.ts

```diff
@@ -0,0 +1,335 @@
+import { Buffer as BufferPolyfill } from 'buffer'
+declare let Buffer: typeof BufferPolyfill
+globalThis.Buffer = BufferPolyfill
+
+import * as asn1js from "asn1js"
+import { bn254, htfDefaultsG1, mapToG1 } from './bn254'
+import { xor } from './utils'
+import { Fp, Fp12, Fp2 } from '@noble/curves/abstract/tower'
+import { AffinePoint } from '@noble/curves/abstract/weierstrass'
+import { createHasher, expand_message_xmd, expand_message_xof, hash_to_field } from "@noble/curves/abstract/hash-to-curve"
```

Comment:
there's a mix of " and ' in the repo - we should align on one and run the formatter